src/lib_ccaligner/phoneme_utils.h

```diff
@@ -16,7 +16,7 @@
 #define CCALIGNER_PHONEME_UTILS_H
 
 #include "commons.h"
-typedef std::string Phoneme;
+using Phoneme = std::string;
```

Comment:
Don't they mean the exact same thing? :) But yeah, this looks cleaner.

Reply:
They are the same, and this looks cleaner as you said :)